packages/gestalt-design-tokens/src/build.js

```diff
@@ -0,0 +1,55 @@
+const StyleDictionary = require('style-dictionary');
+
+const { registerTokenTransformGroups } = require('./transformers/transformGroups');
+const { registerTokenTransforms } = require('./transformers/transforms');
+const { registerTokenFilters } = require('./filters');
+const { getWebConfig } = require('./platforms/web');
+const { getAndroidConfiguration } = require('./platforms/android');
+const { getIOSConfiguration } = require('./platforms/ios');
+const { registerWebFormats } = require('./formatters/registerWebFormats');
+
+StyleDictionary.registerFileHeader({
```

Comment:
shoudl we move this out too to keep everything isolated?

Reply:
Yes - good idea